Makefile

```diff
@@ -181,8 +324,8 @@ $(ENGINE_TARGETS): pyenvinstall
 # test
 # ----
 
-PHONY += test test.sh test.pylint test.pep8 test.unit test.coverage test.robot
-test: buildenv test.pylint test.pep8 test.unit gecko.driver test.robot
+PHONY += test test.sh test.pep8 test.pylint test.unit test.coverage test.robot
+test: buildenv test.pep8 test.pylint test.unit test.robot
```

Comment:
* pylint is slow
* pep8 is fast, and check typo (more than that).

test.pep8 first allow to fail fast on a typo (missing whitespace for example)

Reply:
the tool pep8 does no longer exists, we use the successor 

https://github.com/searx/searx/blob/006f206dc9ff30893cfb6e431c2ee3a7f7768973/requirements-dev.txt#L4

Pycodestyle : identifies pep8 style violations.
Pylint : throws warnings or errors for a combination of programming and style issues.

_pycodestyle is a subset of pylint_

> pep8 is fast, and check typo (more than that).

Only simple typo checks .. [pylint](https://www.pylint.org/) does a deep inspection.

Reply:
I should I have written:
* test.pylint is slow
* test.pep8 (the Makefile target) is fast, and check typo ~(more than that)~.

It is better to fail fast for a typo, then check more in details.